src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationProvider/System/Windows/Automation/Provider/AutomationInteropProvider.cs

```diff
@@ -215,7 +215,7 @@ private static void ValidateArgumentNonNull(object obj, string argName)
         // Throw an argument Exception with a generic error
         private static void ThrowInvalidArgument(string argName)
         {
-            throw new ArgumentException(SR.Format(SRID.GenericInvalidArgument, argName));
+            throw new ArgumentException(SR.Format(SR.GenericInvalidArgument, argName));
```

Comment:
This is bug in the current main branch. SR.Format expects the first parameter to be the resource, **not** the resource name: https://github.com/dotnet/wpf/blob/main/src/Microsoft.DotNet.Wpf/src/Common/src/System/SR.cs#L81-L89

Reply:
@ThomasGoulet73 , what do you mean by 'this is a bug' ? Is the usage here in AutomationInteropProvider.cs wrong ?? Or are you referrring that SR.Format needs to be fixed ??

Reply:
@dipeshmsft Sorry I forgot to answer your question. This specific usage of SR.Format in this file is wrong. Before this PR, `SRID.GenericInvalidArgument` was the key of the resource not the resource itself and SR.Format is a simple wrapper around string.Format. So this code was doing something like this:
`throw new ArgumentException(string.Format("GenericInvalidArgument", argName));`
which is wrong.

Reply:
However, with your PR this is fixed. So, I am marking this as resolved. And thanks for explaining this.